test/functional/apps/management/_scripted_fields.js

```diff
@@ -58,6 +59,7 @@ export default function({ getService, getPageObjects }) {
     before(async function() {
       await browser.setWindowSize(1200, 800);
       // delete .kibana index and then wait for Kibana to re-create it
+      await security.testUser.setRoles(['kibana_admin', 'test_logstash_reader']);
```

Comment:
I think we don't actually need any changes in this file because you're calling setRoles with the default roles.